plugins/janus_streaming.c

```diff
@@ -7565,6 +7574,11 @@ static void *janus_streaming_relay_thread(void *data) {
 					close(source->video_rtcp_fd);
 				}
 				source->video_rtcp_fd = -1;
+
+				if(g_atomic_int_get(&mountpoint->destroyed)){
+					break;
+				}
+				
```

Comment:
No need for brackets here, the action is simple. Extra empty lines are unneeded too.

Reply:
I will del brackets and newline and pull request again.